pyproject.toml

```diff
@@ -57,7 +57,9 @@ dev = [
     "pandas",
 ]
 test = [
+    "aiohttp",
     "awkward>=1.9.0rc1",
+    "fsspec-xrootd",
```

Comment:
If we want uproot to be "batteries included" we should add these to the regular install requirements if we adopt FSSpec as the standard source.

Reply:
See below: I wanted to mainline this discussion because of its large scope.